design/work_item_event.go

```diff
@@ -71,3 +71,24 @@ var _ = a.Resource("work_item_events", func() {
 		a.Response(d.InternalServerError, JSONAPIErrors)
 	})
 })
+
+// user can list an event from the wit service
+// if the event id is given
+var _ = a.Resource("event", func() {
+	a.BasePath("/events")
```

Comment:
I have some questions, on the placement of this endpoint.

if this is placed as `event` as a root level one, how do we know what to return and where to search? The events/revision are of different types, like `comment`, `workitem`, etc.

And all this data is not in one place, there are different tables for different workitem event types. For eg. `work_item_revisions` and `comment_revisions` and `work_item_link_revisions`.

Reply:
cc: @aslakknutsen @baijum